docs/project-charter.md

```diff
@@ -0,0 +1,196 @@
+- Mediating technical conflicts between Collaborators or Foundation projects
+- Hosting and publishing the monthy LoopBack Maintainers Call
+
+### Section 4.1 Project Operations & Management (optional)
+
+_directions: use this section to describe any other specific tasks the_
+_${PROJECT} Governing Body may be responsible for regarding process or project_
+_operations and management. If you are not using this section, please indicate_
+_your intent with the phrase, 'Section Intentionally Left Blank'._
+
+ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob/HEAD/sig-architecture/charter.md#roles-and-organization-management)
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md#section-5-nodejs-project-operations)
+
+The TSC and entire technical community will follow any processes as may be
+specified by the OpenJS Foundation Board relating to the intake and license
+compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+### Section 4.2: Decision-making, Voting, and/or Elections (optional)
+
+_directions: describe any provisions the project makes for decision-making_
+_or include the information by reference your governance.md document._
+_If you are not using this section, please indicate your intent with the_
+_phrase, 'Section Intentionally Left Blank'._
+
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md#section-6-elections)
+
+Unless stated otherwise, the LoopBack TSC adopts a lazy consensus voting system,
+where consensus is assumed when there are no outstanding objections from the
+other TSC members after a stipulated period of time of no less than 72 hours.
+
+For more important matters such as the nomination of new TSC members, a lazy
+consensus for a stipulated period of no less than 1 week is used instead.
+
+These votes must be done through a process that's accessible to the general
+public.
+
+### Section 4.3: Other Project Roles (optional)
+
+_directions: describe other roles within the project, such as chairperson,_
+_tech lead, collaborator, contributor, maintainer, etc. and any responsibilities or_
+_rights such role confers. You can also include this information by_
+_reference to your governance.md document._
+_If you are not using this section, please indicate your intent with the_
+_phrase, 'Section Intentionally Left Blank'._
+
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md#section-8-project-roles)
+
+The role of a LoopBack Maintainer is given to those that the the LoopBack TSC
+has confidence in their general understanding of the Project's codebase, and
+trusts that they are capable for seeking feedback and consensus on non-trivial
+contributions. This role provides permissions to merge pull requests in the
+Project's Git Repositories, and dedicated communication channels with the
+LoopBack TSC. Nomination for the role is done in accordance with
```

Comment:
We have a dedicated channel for the maintainers. I assume that's what you're referring to?

```suggestion
rest of the LoopBack maintainer community. Nomination for the role is done in accordance with
```